WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "4.31.0"
+  s.version       = "4.31.1-beta.1"
```

Comment:
`4.31` has already been cut; we're now on `4.32`. Also, the previous person didn't update the version correctly so it's hard to tell, but there is [already](https://github.com/wordpress-mobile/WordPressKit-iOS/releases) a `4.32.0-beta.1` so this should be `4.32.0-beta.2`.

Reply:
Making a note to check the releases in the future rather than looking at the podspec. :+1: